Ruddarr/Views/Movies/MovieDetails+Information.swift

```diff
@@ -22,6 +22,10 @@ extension MovieDetails {
     }
 
     var informationItems: [InformationItem] {
+        let tagLabels = movie.tags.compactMap { tagId in
+                    instance.tags.first { $0.id == tagId }?.label
```

Comment:
We might want to fall back to `tag:<id>` as the string, instead of omitting it, so it's not misleading if the tag is new. Same for series.